### ISSUE TITLE
Fetch logo.png from github

### ISSUE THREAD

**Issue by dmi3kno** [akgold/stickr#1]
Many packages specify github repo in their Description file. You could use github api to contact the repo and try and fetch logo.png from `man/figures/logo.png`. Here's one for tidyverse:
![tidyverse logo](https://raw.githubusercontent.com/tidyverse/tidyverse/master/man/figures/logo.png)

**Comment by akgold** [comment]
That's an interesting idea -- so in that case, someone could specify a `user/repo` combo and we could try to fetch the sticker?

**Comment by alexkgold** [comment]
Huh, that's a cool idea. 

Seems like the issue is that the github URLs are included inconsistently...for example `textmineR` has the github URL only in the bug report section, and then at`/issues`. But that's solvable with some regex...

What do you think the flow here would look like from a user perspective?

Right now, someone can see all available with `stickr_list` and then use a getting function to get them.

If arbitrary packages were allowed, someone could drop in the name of a package installed on the system into a getter function, if it's not in `stickr_list`, it'd be delivered if possible? One issue I foresee is that the newest logo isn't always what's under `logo.png` (a la textminer). Should people be able to specify where the logo image is in the repo? Or should it require a fully-specified repo (`TommyJones/textmineR`)?

Maybe the right answer is to try, but allow people to pass in values if the search doesn't work.

**Comment by dmi3kno** [comment]
You don't need a list and you shouldnt make a guarantee. The package "tries to" retrieve a logo.
I see the algorithm as follows:
```
1: For arbitrary package name provided by user
2:   get package repo
3:   get content of `man/figures`
4:   if logo.[png|jpg|bmp|gif] is present, **return** it, otherwise
5:   get README.md, find header (should match package name)
6:   in immediate proximity to it, there should be reference to <img> in `man/figures` displayed not larger than 300x300 (either image size or rescaled at display in HTML tag). Return it. 
```
For 2: have a look at https://github.com/yonicd/helpdesk. He is parsing DESCRIPTION file searching for references to README or pkgdown site. Maybe you could rely on his results and just look for `logo*.[png|jpg|bmp|gif]` and just return it, as a first try.

**Comment by alexkgold** [comment]
Cool, thanks for feedback -- this is mostly implemented as of 0.2.0, which I just pushed. This isn't *fully* implemented, as I basically only look in `man/figures` for the time being. If it turns out that there are a lot of stickers elsewhere, maybe I'll add.
